README.md

```diff
@@ -13,76 +13,33 @@ Once created, this project can be easily and quickly deployed to Vercel from the
-```
-Congratulations! Your starter project is now live and can be viewed at [http://localhost:3000](http://localhost:3000).
-
-If you use yarn, you might want to delete `package-lock.json` file to avoid possible  resolution inconsistencies caused by unsynchronized lock files.
 
 ## 2) Set API Token
 
-For Next.js to be able to fetch your ButterCMS content properly, you need to add your API token as an environment variable. To do that, copy `env.sample` and edit it, so that includes your API token.
+To fetch your ButterCMS content, add your API token as an environment variable.
 
 ```bash
-$ cp env.sample .env
-```
-```
-# .env
-NEXT_PUBLIC_BUTTER_CMS_API_KEY=<Your API Token>
+$ echo 'NEXT_PUBLIC_BUTTER_CMS_API_KEY=<Your API Token>' >> .env
 ```
 
 ## 3) Run the local server
-You'll want to run server any time you want to view the project locally:
 
-```bash
-npm run dev # or yarn dev
-```
-
-To quit the server press CONTROL-C.
-
-### Build the app
-Use one of these commands which build the application for production usage:
+To view the app in the browser, you'll need to run the local development server:
 
 ```bash
-npm run build # or yarn build
-```
-
-### Run production server
-After building your app run one of these commands to start a Next.js production server:
-
-```bash
-npm run start # or yarn start
-```
-
-## 4) Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-1. Create a Vercel account at https://vercel.com/signup
-2. Download [the CLI](https://vercel.com/download):
-```bash
-npm i -g vercel
+npm run dev # or yarn dev
 ```
-3. Add the API key as a secret `vercel secrets add butter-cms-api-key "YOUR_API_KEY"`
-4. Run `vercel` at the project root
-
-## Webhooks to trigger deployment
 
-Use of the ButterCMS webhooks to notify the application every time a post, page or collection item are added or modified, and reload the Next.js application.
+Congratulations! Your starter project is now live: [http://localhost:3000](http://localhost:3000).
 
-You can add a new webhook by going to the [https://buttercms.com/webhooks/](https://buttercms.com/webhooks/) page.
+## 4) Deploy
 
-![webhook page](https://cdn.buttercms.com/ixt8yfOsSgWLlwc0eis1)
+Deploy your Butterized proof of concept app and to spread your love of Butter.
 
-If you use Vercel, you can create deploy webhooks there. ![Vercel deploy webhooks](https://cdn.buttercms.com/bPIzuIXTleK7h4Kx3X4L)
+###  Vercel
 
-Learn more about webhooks [here](https://buttercms.com/docs/api/#webhooks).
+Deploy your Next.js app using Vercel the creators of Next.js. This process will create a copy of the this starter project in your Git provider account, instantly deploy it, and have a full content workflow connected to your ButterCMS account. Smooth.
```

Comment:
```suggestion
Deploy your Next.js app using Vercel, the creators of Next.js. With the click of a button, you'll create a copy of your starter project in your Git provider account, instantly deploy it, and institute a full content workflow connected to your ButterCMS account. Smooth.
```

These are little nitpicky things - obviously the wording is up for grabs, but there were some places where the subject/verb agreement weren't super clear, so I just went in and tweaked those.